com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageHandler.cs

```diff
@@ -227,42 +205,19 @@ internal static void HandleAddObject(ulong clientId, Stream stream)
 #endif
             using (var reader = PooledNetworkReader.Get(stream))
             {
-                bool isPlayerObject = reader.ReadBool();
-                ulong networkId = reader.ReadUInt64Packed();
-                ulong ownerClientId = reader.ReadUInt64Packed();
-                bool hasParent = reader.ReadBool();
+                var isPlayerObject = reader.ReadBool();
+                var networkId = reader.ReadUInt64Packed();
+                var ownerClientId = reader.ReadUInt64Packed();
```

Comment:
```suggestion
                var networkObjectId = reader.ReadUInt64Packed();
```

Reply:
Yeah, I noticed the names but decided to leave them out for this PR.  This area will get reworked.  For now, I am going to keep all of the names the same as it s used in several places.